chainer/iterators/order_samplers.py

```diff
@@ -0,0 +1,46 @@
+import numpy
+
+
+class ShuffleOrderSampler(object):
```

Comment:
Let's add class documentation (maybe you were going to later?) describing also that it's called once per epoch etc.